src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const cloneState = JSON.parse(JSON.stringify(state));
```

Comment:
could have been simpler
```suggestion
  const cloneState = { ...state };
```